crawler/plugins/systems/fprobe_container_crawler.py

```diff
@@ -145,6 +145,8 @@ def start_fprobe(self, ifname, user, bindaddr, port, **kwargs):
                   '-n', '%s:%d' % (bindaddr, port)]
         if len(fprobe_bpf.strip()):
             params.insert(1, fprobe_bpf)
+        if netflow_version == 10:
+            params.insert(1, '-b')
```

Comment:
is this a standard flag? or works with the specific softflowd version part of this PR?